interface/vmck.py

```diff
@@ -34,21 +34,24 @@ def evaluate(submission):
     options = vmck_config(submission)
     name = f'{submission.assignment.full_code} submission #{submission.id}'
     options['name'] = name
-    options['manager'] = True
+    options['restrict_network'] = True
+
+    options['manager'] = {}
+    options['manager']['vagrant_tag'] = settings.MANAGER_TAG
+    options['manager']['memory'] = settings.MANAGER_MEMORY
```

Comment:
Q: This is how much memory/cpu a virtual machine will use, right? (the machine where the student homework will run, right?)

Reply:
A: Not quite. There is another task that runs alongside the student's vm (it uses [ vmck/vagrant-vmck](https://github.com/vmck/vagrant-vmck) -- the submission folder). This connects to the vm, runs the scripts and eventually sends back the results, stdout and stderr to acs-interface. These are its resources (50 Mb - of memory and 30 Mhz of cpu if i am not mistaken).